src/book/02-system/02-subsystems/01-input.mdx

```diff
@@ -6,7 +6,7 @@ description: How inputs are handled in the NUbots codebase.
 slug: /system/subsystems/input
 ---
 
-Input to the system includes cameras, Game Controller and NatNet.
+Input to the system includes cameras, [Game Controller](https://github.com/RoboCup-Humanoid-TC/GameController) and [NatNet](https://optitrack.com/products/natnet-sdk/).
```

Comment:
Does this change really belong in this PR?

Reply:
The whole page, or just the specific change?

Reply:
Yes.